src/components/scenes/LoginScene.tsx

```diff
@@ -82,7 +82,7 @@ class LoginSceneComponent extends React.PureComponent<Props, State> {
       // @ts-expect-error
       .catch(e => this.setState({ backgroundImage: theme.backgroundImage }))
     const { YOLO_USERNAME, YOLO_PASSWORD, YOLO_PIN } = ENV
-    if (YOLO_USERNAME != null && (YOLO_PASSWORD != null || YOLO_PIN != null) && firstRun) {
+    if (!!YOLO_USERNAME && (!!YOLO_PASSWORD || !!YOLO_PIN) && firstRun) {
```

Comment:
Not approved. Do the `Boolean()` thing instead. We eventually want to enable a lint rule that disallows treating strings and other data types as booleans, such as using `!` or `if` or `?`. The `Boolean()` function encapsulates the `!= null && !== ''` logic in a way ESLint will be happy with.